### ISSUE TITLE
revokeDeviceLicense endpoint 400 Bad Request

### ISSUE THREAD

**Issue by cltj** [microsoftgraph/microsoft-graph-docs#10164]
This endpoint does not seem to respond when called. 

POST https://graph.microsoft.com/beta/deviceAppManagement/mobileApps/{{mobileAppId}}/revokeDeviceLicense

![revokeFail01](https://user-images.githubusercontent.com/72233033/95012780-98b88a00-063b-11eb-8a80-3dbbb34c2e23.png)

In addition I would love it if there was some information on the page about these variables and where to find them. 
* **{mobileAppInstallStatusId}** 
* **{userAppInstallStatusId}** 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2dbfc6be-6843-d2b6-798a-14194a83b52f
* Version Independent ID: 6d4712b4-a052-6735-ee92-8534ec76931f
* Content: [revokeDeviceLicense action - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/intune-apps-iosvppapp-revokedevicelicense?view=graph-rest-beta)
* Content Source: [api-reference/beta/api/intune-apps-iosvppapp-revokedevicelicense.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/intune-apps-iosvppapp-revokedevicelicense.md)
* Product: **intune**
* Technology: **microsoft-graph**
* GitHub Login: @dougeby
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by cltj** [comment]
Any updates on this one?

**Comment by peterson-dane** [comment]
@cltj The URI, as presented in the docs, doesn't work. It'll always give you an error. Eventually, while searching around, I found someone who documented a script of theirs that did work, and I found this format for the path in the URI (I've highlighted the addition):

```
                                              ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
/deviceAppManagement/mobileApps/{mobileAppId}/microsoft.graph.iosVppApp/revokeDeviceLicense
```

I've used this many times since via the script that I was working on at the time, so I can confirm that it works. I've looked through the docs a fair amount on this one, both before and after finding a solution, trying to determine where I might have missed something, and I've not been able to.

Nowhere else that I've used any kind of GUID when working with the Intune API have I needed to specify the OData type as part of the URI.

I think the docs probably need to be edited to include this as part of the path(s) listed. I recall trying all (or nearly all) of the listed paths, and they all gave some kind of 400 or 404 error at the time.
